package multipleException;

import java.io.IOException;

public class App {
	public static void main(String args[]){
	Multi multi1 = new Multi();
	try {
		multi1.run();
	} catch (IOException e) {
		// TODO Auto-generated catch block
		System.out.println(e.getMessage());
	}
	}

}
